pkg/tangy/rpm.go

```diff
@@ -0,0 +1,52 @@
+package tangy
+
+import (
+	"context"
+	"strings"
+
+	"github.com/jackc/pgx/v5"
+)
+
+type RpmPackageSearch struct {
+	Name    string
+	Summary string
+}
+
+func (t *tangyImpl) RpmRepositoryVersionPackageSearch(ctx context.Context, hrefs []string, search string) ([]RpmPackageSearch, error) {
```

Comment:
i think we want this to take a limit, so we can expose that limit to the user like we do for rpms